### ISSUE TITLE
WCAG Failure: Selecting the Information, Copyrights, or Index Tabs screen reader focus is sent to the top of the page.

### ISSUE THREAD

**Issue by patdunlavey** [ProjectMirador/mirador#3918]
This issue description is from an external accessibility audit of our Mirador 3 install
- Selecting the Information, Copyrights, or Index Tabs, screen reader focus is sent to the top of the page forcing screen reader users to navigate back through the page. Same issue happens when a page is selected. 
- Failed WCAG Success Criteria: 3.2.2 On Input (A)
- Recommendation: Keep the focus on the activating element when triggered.
